pkg/kubelet/config/flags.go

```diff
@@ -86,7 +86,8 @@ func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 	dockerOnlyWarning := "This docker-specific flag only works when container-runtime is set to docker."
```

Comment:
can you please update this text as well as part of this PR since this message mention the flag you change.

Reply:
@SergeyKanzhelev this line is being removed as a part of dockershim flag removal PR
https://github.com/kubernetes/kubernetes/pull/106907/files#diff-5dd381e5c058ed2af7d72be5f6d1b540bdba3a5193eaa069214786f8bdf535cfL86